docs/api-reference.md

```diff
@@ -83,14 +83,14 @@ error_type_enum = EnumType("ErrorType", ErrorType)
 FallbackResolversSetter()
 ```
 
-[_Bindable_](bindables.md) used for setting default resolvers on schema object types.
+A [_bindable_](bindables.md) used for setting the default resolvers on the schema object types.
```

Comment:
I don't think that `the` before `default resolvers` is needed here - we are not definitive in this place.